src/main.rs

```diff
@@ -171,7 +171,7 @@ fn run_spec(spec_text: &str, args: &[String], opts: Options) {
             print!(" ");
         }
 
-        let our_html = render_html(&test.input.replace("→", "\t"), opts);
+        let our_html = render_html(&test.input.replace("→", "\t").replace("\n", "\r\n"), opts);
```

Comment:
Will this leave `\r\r\n` if the input file already had CRLF endings? I wouldn't call it "normalizing" unless it will have the same behavior either way.